docs/resources/ces_alarm_template.md

```diff
@@ -0,0 +1,105 @@
+---
+subcategory: "Cloud Eye (CES)"
+---
+
+# huaweicloud_ces_alarm_template
+
+Manages a CES alarm template resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "name" {}
+
+resource "huaweicloud_ces_alarm_template" "test"{
+  name = var.name
+
+  policies {
+    namespace           = "SYS.APIG"
+    dimension_name      = "api_id"
+    metric_name         = "req_count_2xx"
+    period              = 1
+    filter              = "average"
+    comparison_operator = ">="
+    value               = "10"
+    unit                = "次/分钟"
+    count               = 3
+    alarm_level         = 2
+    suppress_duration   = 43200
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the name of the CES alarm template.
+  An alarm template name starts with a letter or Chinese, consists of 1 to 128 characters,
+  and can contain only letters, Chinese characters, digits, hyphens (-) and hyphens (-).
+
+* `policies` - (Required, List) Specifies the policy list of the CES alarm template.
+The [Policy](#CesAlarmTemplate_Policy) structure is documented below.
+
+* `description` - (Optional, String) Specifies the description of the CES alarm template.
+  The description can contain a maximum of 256 characters.
+
+<a name="CesAlarmTemplate_Policy"></a>
+The `Policy` block supports:
+
+* `namespace` - (Required, String) Specifies the namespace of the service.
+
+* `dimension_name` - (Required, String) Specifies the Resource dimension.
+  The name starts with a letter and separated by commas(,) for multiple dimensions,
+  can contain only letters, digits, hyphens (-) and hyphens (-),
+  and contain a maximum of 32 characters for each dimension.
+
+* `metric_name` - (Required, String) Specifies the alarm metric name.
+
+* `period` - (Required, Int) Specifies the judgment period of alarm condition.
+  Value options: **1**, **300**, **1200**, **3600**, **14400**, **86400**.
+
+* `filter` - (Required, String) Specifies the data rollup methods.
+  Value options: **average**, **variance**, **min**, **max**, **sum**.
+
+* `comparison_operator` - (Required, String) Specifies the comparison conditions for alarm threshold.
+  Value options: **>**, **<**, **=**, **>=**, **<=**.
+
+* `value` - (Required, Int) Specifies the alarm threshold.
+
+* `unit` - (Optional, String) Specifies the unit string of the alarm threshold.
+The unit can contain a maximum of 32 characters.
+
+* `count` - (Required, Int) Specifies the number of consecutive triggering of alarms.  The value ranges from 1 to 5.
+
+* `level` - (Optional, Int) Specifies the alarm level. Value options:
```

Comment:
add  a description of the default value

Reply:
there is no default value, but add an explain